src/bun.js/bindings/BunProcess.cpp

```diff
@@ -1351,7 +1351,9 @@ static JSValue constructReportObjectComplete(VM& vm, Zig::GlobalObject* globalOb
 
         {
             char cwd[PATH_MAX] = { 0 };
-            getcwd(cwd, PATH_MAX);
+            char *r = getcwd(cwd, PATH_MAX);
+
+	    assert(r);
```

Comment:
If this is mutating `cwd` then I don't think we need to assign it to a variable. We can add a comment that `getcwd` mutates.

Reply:
`getcwd` does mutate, but returns `NULL` on error. We could change to `assert(getcwd(cwd, PATH_MAX))`, but I don't see an issue with the explicit version.